distribution/src/config/opensearch.yml

```diff
@@ -8,7 +8,7 @@
 # the most important settings you may want to configure for a production cluster.
 #
 # Please consult the documentation for further information on configuration options:
-# https://www.elastic.co/guide/en/elasticsearch/reference/index.html
+# https://www.opensearch.org
```

Comment:
Probably add a TODO to update the URL when Docs link is available ?

Reply:
> Probably add a TODO to update the URL when Docs link is available ?

Tianli or Himanshu might have know what is corresponding think that OpenSearch is using?

Reply:
What's the reasoning behind this change? Please see my comment on issue #609